com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/font/Fontc.java

```diff
@@ -739,8 +765,6 @@ else if (channelCount==3)
                 throw new FontFormatException("Could not generate font preview: " + e.getMessage());
             }
         }
-        boolean is_monospaced = true;
-        float base_advance = include_glyph_count > 0 ? glyphs.get(0).advance : 0;
```

Comment:
The issue with the previous code is that we looped over the _output_, which may contain only 1 (and ultimately 0) glyphs, so it was always determined as "monospaced".